platform-includes/getting-started-config/javascript.react.mdx

```diff
@@ -56,6 +56,29 @@ root.render(<App />);
 
 Once this is done, all unhandled exceptions are automatically captured by Sentry.
 
+### React 19 Error Reporting
+
+Starting with React 19, the `createRoot` and `hydrateRoot` methods from `react-dom` expose error hooks that can be used to capture errors automatically. Use the `Sentry.reactErrorHandler` function to capture errors in the error hooks you are interested in.
```

Comment:
You say that the hooks are used to capture errors automatically and then describe how to use the handler function to capture errors in the error hooks of interest... I'm confused if users actually have to do anything with the sample below or if it's automatically captured for them? Maybe this revision can help clarify?

```suggestion
Starting with React 19, the `createRoot` and `hydrateRoot` methods from `react-dom` expose error hooks that are used to capture errors automatically. If you want to customize how errors are handled in specific error hooks, you can use the `Sentry.reactErrorHandler` function.
```

Reply:
I agree with your change. Users do have to manually set the hooks themselves, so this makes it more clear!